### DISCUSSION TITLE
Increase pvalue cutoff for motifmatchr when running AddMotifs?

### DISCUSSION THREAD

**Discussion by owenwilkins** [stuart-lab/signac#1427] [General]
Is there a way to increase the `p.cutoff` argument of `motifmatchr::matchMotifs()` when using `AddMotifs()` in signac. 

The default value of 5e-5 is quite low and I expect generates quite a lot of false positives, causing the number of enriched motifs to be inflated when running `FindMotifs`.

**Comment by timoast** [comment]
You can create the `Motif` object yourself using the `CreateMotifObject()` function, this will allow you to run the `matchMotifs()` function yourself (or any other function to create the motif position matrix). See this code for example: https://github.com/stuart-lab/signac/blob/2ad6c3c9c0c8dd31f7e1433b2efd5050d8606f27/R/motifs.R#L32-L59

**Comment by owenwilkins** [comment]
fantastic, thanks
